projects/igniteui-angular/src/lib/core/styles/components/action-strip/_action-strip-theme.scss

```diff
@@ -138,22 +138,6 @@
             width: rem(18px);
             height: rem(18px);
         }
-
-        igx-icon + [igxLabel] {
```

Comment:
These changes are not connected to the button refactoring, they will break the dropdown items inside the action strip. see the [screenshot](https://imgur.com/3ajKm1D)